src/components/integrated/OakLessonVideoTranscript/__snapshots__/OakLessonVideoTranscript.test.tsx.snap

```diff
@@ -88,13 +88,13 @@ exports[`OakLessonVideoTranscript matches snapshot 1`] = `
 
 .c7 {
   font-family: Lexend,sans-serif;
-  font-weight: 700;
-  font-size: 1.125rem;
-  line-height: 1.75rem;
-  -webkit-letter-spacing: -0.005rem;
-  -moz-letter-spacing: -0.005rem;
-  -ms-letter-spacing: -0.005rem;
-  letter-spacing: -0.005rem;
+  font-weight: 600;
+  font-size: 1rem;
+  line-height: 1.25rem;
+  -webkit-letter-spacing: 0.0115rem;
+  -moz-letter-spacing: 0.0115rem;
+  -ms-letter-spacing: 0.0115rem;
+  letter-spacing: 0.0115rem;
```

Comment:
rug was pulled on this in a previous merge so main was broken. This fixes the broken snapshot